airflow-core/tests/unit/serialization/serializers/test_serializers.py

```diff
@@ -166,12 +187,60 @@ def test_encode_k8s_v1pod(self):
             "spec": {"containers": [{"image": "bar", "name": "foo"}]},
         }
 
+    def test_bignum(self):
+        from airflow.serialization.serializers.bignum import serialize
+
+        assert serialize(12345) == ("", "", 0, False)
```

Comment:
This should probably be named better, such as `test_bignum_not_decimal` or something?